### DISCUSSION TITLE
deepin-terminal: [Feature Request] Change Bing to DuckDuckGo in dropdown menu

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#2927] [Features Request & Ideas | 特性请求 & 头脑风暴]
I feel like DuckDuckGo would be a more appropriate choice especially for a Linux distribution. It's the most privacy focused and popular search engine in the Linux world. If you just wanted to put the most popular search engine overall, google is better anyways.

**Comment by ch593030323** [comment]
Yeah, i agree with ur idea that Google Engine is more poplar. 
We will think about adding this search engine later.
U can just replace "https://cn.bing.com/search?q=" by  the "google search engine url".

**Comment by raspin0** [comment]
That's a tricky situation right there. I'm personally using DuckDuckGo, but I agree that providing Google search would cover the biggest percentage of people from EU/NA. However, since additionally there is a need to serve Baidu for China I wonder if it's possible to make an universal search to fulfill everyone's preferences.
I'm thinking about a feature that would take selected query from the terminal and pass it into the browser which would just use it's default search engine.

**Comment by ch593030323** [comment]
Thank you for your suggestion. We will consider this function reasonably.
